recipe_scrapers/coopse.py

```diff
@@ -0,0 +1,110 @@
+# mypy: allow-untyped-defs
+import json
+import re
+
+import requests
+
+from recipe_scrapers._grouping_utils import IngredientGroup
+
+from ._abstract import AbstractScraper
+
+SCRIPT_PATTERN = re.compile('"recipeId":"(?P<recipeId>[0-9]+)"')
+
+
+class CoopSE(AbstractScraper):
```

Comment:
Pedantic note: in a previous case (#927) I suggested removing a top-level-domain suffix from the scraper name.  In this case, I think we should keep the `SE` suffix, because coop / co-op exists as a locally-recognized name, referring to different entities, in multiple regions of the world.

Reply:
Yes, I wasn't sure what the convention was. I checked some other coop-sites and unfortunately they used a different system for recipes, so different scrapers would be needed.